Dev/VideoChatClient/VideoChatClientException/VideoChatClientException.hpp

```diff
@@ -0,0 +1,41 @@
+#pragma once
+#include <cstdint>
+#include <ostream>
+#include <exception>
+#include "VideoChatClientStatus.hpp"
+
+class VideoChatClientException : public std::exception
+{
+	friend std::ostream& operator<<(std::ostream& os, const VideoChatClientException& obj)
+	{
+		return os
+			<< "VideoChatClientException{status: " << obj.m_status
+			<< " error: " << obj.m_last_error << "}";
+	}
+
+public:
+	VideoChatClientException(VideoChatClientStatus status, uint32_t last_error) :
+		std::exception(), m_status(status), m_last_error(last_error)
+	{
+	}
+
+	~VideoChatClientException() override = default;
```

Comment:
this declaration is not used, we do not implement any logic in the D'tor, so why override it?